### PR TITLE
Added missing mech properties

### PR DESCRIPTION
**Pull request by nissimsan**
MTR was missing mech property examples

### CODE REVIEW

File: docs/openapi/components/schemas/credentials/MillTestReportCertificate.yml [hunk 1]
```diff
@@ -478,8 +478,8 @@ example: |-
                 "type": [
                   "MechanicalProperty"
                 ],
-                "identifier": "",
-                "name": "Gauge Length"
+                "identifier": "ISO 1352:2021",
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/credentials/MillTestReportCertificate.yml:481]: https://github.com/w3c-ccg/traceability-vocab/blob/e92bb49a71b74c3d05fe75dc76ef1e99fdde3d80/docs/openapi/components/schemas/common/SteelProduct.yml#L119
These seem to have a minor difference in them, probably a good idea for them to be identical

File: docs/openapi/components/schemas/credentials/MillTestReportCertificate.yml [hunk 2]
```diff
@@ -497,7 +497,7 @@ example: |-
                 "type": [
                   "MechanicalProperty"
                 ],
-                "identifier": "",
+                "identifier": "ISO 1519:2011",
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/credentials/MillTestReportCertificate.yml:500]: This one does not exist on the steel product

**Review by mkhraisha — CHANGES_REQUESTED** [review]

**Comment by brownoxford** [comment]
Consensus on call to fix and merge when ready

**Comment by nissimsan** [comment]
I'll remove the package lock before merging

**Comment by nissimsan** [comment]
@mkhraisha, I've copied the examples from SteelProduct now instead as you suggest. Made a separate PR to avoid git acrobatics: https://github.com/w3c-ccg/traceability-vocab/pull/580

**Pull request closed without merging**